src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends ColorSupplier implements Figure {

```

Comment:
lets use out ColorSupplier in FigureSupplier class(and remove it from all figures classes), and generate Random colour for each figure which is created there